_pages/security/incident-response-drills.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Incident Response Drills
+sidenav: security
+sticky_sidenav: true
+---
+
+*Table of Contents*
+
+* [Why do Incident Response Drills?](#why-do-incident-response-drills)
+* [How to Build Incident Response Drills](#how-to-run-an-incident-response-drill)
+* [Example Incident Response Drills](#example-incident-response-drills)
+* [Using this drill as part of your ATO](#congratulations-you-accidentally-did-compliance-too)
+
+## Why do Incident Response Drills?
+
+You don't want to be creating or testing recovery processes while things are on fire. 🔥
```

Comment:
I love it

Reply:
```suggestion
You don't want to be creating or testing recovery processes while things are on fire. 🔥

When things are on fire, you want to be able to focus on fixing the issues and getting your system back online and in working order as soon as possible. Having an already established incident recovery practice will allow you and your team to focus on the problem, rather than the process.
```